lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -43,7 +43,9 @@ class InfoCommand < HammerCLIForeman::InfoCommand
           Fields::Field.new(:label => _('Server'), :path => [:server])
         ],
         'openstack' => [
-          Fields::Field.new(:label => _('Tenant'), :path => [:tenant])
+          Fields::Field.new(:label => _('Tenant'), :path => [:tenant]),
+          Fields::Field.new(:label => _('Project domain name'), :path => [:project_domain_name]),
+          Fields::Field.new(:label => _('Project domain ID'), :path => [:project_domain_id])
```

Comment:
thanks @lzap, Looks good :)
If you are already changing this code, I think It would be better to have in each compute resource file, a function for 'provider_spesific_fileds'  so that compute resource plugins will able to add their specific details easily

for example: 

**/hammer-cli-foreman/lib/hammer_cli_foreman/compute_resource/base.rb**
will contain:
`def provider_specific_fields; []; end
`
**/hammer-cli-foreman/lib/hammer_cli_foreman/compute_resource/openstack.rb** 
will contain:

```
def provider_spesific_fileds
   [
          Fields::Field.new(:label => _('Tenant'), :path => [:tenant]),
          Fields::Field.new(:label => _('Project domain name'), :path => [:project_domain_name]),
          Fields::Field.new(:label => _('Project domain ID'), :path => [:project_domain_id])
    ]
end 
```


and the print data function in **compute_resource.rb** will look like that:

```
def print_data(data)
        provider = ::HammerCLIForeman.compute_resources[data["provider"].downcase]
        if provider
          output_definition.fields.concat(provider.provider_specific_fields || [])
          super(data)
        end
 end
```

Reply:
@lzap, @shiramax,

Added suggested changes into [PR-431](https://github.com/theforeman/hammer-cli-foreman/pull/431)